### PR TITLE
Carson county/Quad County Region, NV

### PR DESCRIPTION
**Pull request by firebuggirl**
Hi all,

This is a simple scraper for the Carson county/Quad County Region sourced from Carson City Health and Human Services, which has aggregated data for the Quad County region of Nevada: Carson City, Douglas, Lyon, and Storey counties.

I wasn't sure what the priority should be set to (as to not get in the way of any other data sources of higher priority), so I left it at the default of `0`. If there is anything that I need to change to avoid conflict(s), let me know.

Cheers:-)

PS. Please ignore the Docker scripts that I added to the package.json via my local fork of this repo. ..not sure how that got into this PR on the branch `carson-county`. I have created a separate branch (apart from this one) to run this project via Docker locally and did not intend to have those changes posted here. Apologies for the mistake and for the confusion.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,8 @@
     "dateData": "DATE=$(date +'%Y-%m-%d') && cp dist/data.json dist/data-$DATE.json && cp dist/features.json dist/features-$DATE.json && cp dist/data.csv dist/data-$DATE.csv",
     "deploy": "npm run dateData && npm run deploySite",
     "deploySite": "npm run buildSite && gh-pages -a -d dist/ -e .",
+    "buildDocker": "docker build -t corona-nginx-html-server:v1 . && docker run -d -p 8080:8080 corona-nginx-html-server:v1 nginx",
+    "stopDockerContainer": "docker container stop $(docker ps -q --filter ancestor=corona-nginx-html-server:v1 )",
```
Comment (lazd) [inline package.json]:
```suggestion
```

**Comment by lazd** [comment]
Nice work! So tough call on Carson City -- while clearly an independent city, it is a county-level entity. We can leave it as a `county` for now and re-assess, maybe with the help of someone from Nevada who has an opinion.

**Comment by lazd** [comment]
I patched up a couple tiny things, see the commits. This is an excellent and welcome contribution! Thank you!

**Comment by firebuggirl** [comment]
Thanks! I will check out the commits. I know, I struggled with the idea of Carson City being both a city and a county-level entity... I decided to give it a go anyway, just to see if it could help in some way:-)

**Pull request merged**